eclipse/tern.eclipse.ide.server.nodejs.core/src/tern/eclipse/ide/server/nodejs/core/NodejsDebuggersManager.java

```diff
@@ -0,0 +1,47 @@
+/**
+ *  Copyright (c) 2015 Genuitec LLC.
+ *  All rights reserved. This program and the accompanying materials
+ *  are made available under the terms of the Eclipse Public License v1.0
+ *  which accompanies this distribution, and is available at
+ *  http://www.eclipse.org/legal/epl-v10.html
+ *
+ *  Contributors:
+ *  Piotr Tomiak <piotr@genuitec.com> - initial API and implementation
+ */
+package tern.eclipse.ide.server.nodejs.core;
+
+import java.util.Collection;
+import java.util.Collections;
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+import tern.eclipse.ide.server.nodejs.internal.core.debugging.webclipse.WebclipseDebugger;
+
+public class NodejsDebuggersManager {
+
+	private static Map<String, INodejsDebugger> debuggers = new LinkedHashMap<String, INodejsDebugger>();
+
+	static {
+		// Add supported debuggers
+		addDebugger(new WebclipseDebugger());
```

Comment:
>  I don't see any immediate gains and I've already spend some time to get this done.

Yes @piotrtomiak I know. I would like just to know the opinion of other guys. If they prefer having an extension point, I will do that, no problem.

> However, it is important for us to have Webclipse displayed, even if it is not installed. 

Yes I understand. My idea is to keep your code (with Uninstalled, etc) and move to a dedicated plugin. In your case, you will just add this plugin to MyEclipse to have every time Webclipse debugger.

To be honnest with you, I don't know if JBoss guys will be OK to have Webclipse debugger inside JBoss Studio, it's just my fear. 

In other words I don't want to have trouble with JBoss guys and you. I want to find the best solution for JBoss and MyEclipse. Hope you will understand and sorry to make you waste your time.